docs/signals/attributes/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Defining Attributes"
+sidebar_position: 20
+description: "Attributes."
+sidebar_label: "Attributes"
+---
+
+Attributes are the building blocks of Snowplow Signals. They represent specific facts about user behavior and are calculated based on events in your Snowplow pipeline. This guide explains how to define an `Attribute`, and use `Criteria` to filter events for precise data aggregation.
+
+- **Number of Pageviews in the Last 7 Days:** Counts how many pages a user has viewed within the past week.
+- **Last Product Viewed:** Identifies the most recent product a user interacted with.
+- **Previous Purchases:** Provides a record of the user's past transactions. 
+
+### Basic Usage
+
+An `Attribute` can be defined to count the number of pageviews in a session through the Python SDK as follows:
+
+```python
+from snowplow_signals import Attribute, Event
+
+page_views_attribute = Attribute(
+    name='page_views_count',
+    type='int32',
+    events=[
+        Event(
+            vendor="com.snowplowanalytics.snowplow",
```

Comment:
Why wouldn't this be snowplow.io?

Reply:
The page_view event was created before we changed to .io, so that's the correct vendor for in iglu central.